src/containers/Bookings.js

```diff
@@ -1,19 +1,23 @@
-import React, { Component } from 'react';
-import Search from '../components/Search.js';
-// import Results from '../components/Results.js';
-// import FakeBookings from '../data/fakeBookings.json';
-
+import React, { Component } from "react";
+import Search from "../components/Search.js";
+import Results from "../components/Results.js";
+import rsults from "../data/fakeBookings.json";
 export default class Bookings extends Component {
+  state = { rsults: undefined };
   search = () => {
-    console.info('TO DO!');
+    this.setState({ rsults});
```

Comment:
👍 We're going to flesh out this function more in upcoming exercises once we've looked at state in class.